### PR TITLE
update readme

### LINKED ISSUE #223: Make proper README with list of functionalities & install instructions (?)

### PR DESCRIPTION
**Pull request by wattsecs**
Update the readme to be ready for the submission deadline.
TODO:
- [x] Add instructions for the key secret

### CODE REVIEW

**Comment by bbjubjub2494** [comment]
> * [ ]  Add instructions for the key secret

I sent an e-mail to the TAs with a copy so as far as they are concerned we should be good. This is still worth mentioning for 3rd parties.

> Please find attached a cleartext copy of the build secrets you will need to build the HumanLearning app next week. It is sufficient to leave the file at the root of the repository.

File: README.md [hunk 1]
```diff
@@ -5,3 +5,30 @@
 
 ## Test Coverage
 [![Test Coverage](https://api.codeclimate.com/v1/badges/88c3d9e8784c8fa76af6/test_coverage)](https://codeclimate.com/github/HumanLearning2021/HumanLearningApp/test_coverage)
```
Comment (bbjubjub2494) [inline README.md:5]: While we're at it, we don't need a dedicated heading for each badge, they are designed to go right under the title. see e.g. [this open source project](https://github.com/arrow-kt/arrow#readme)

**Review by NicolasVial — APPROVED** [review]
I think it's pretty clear and enough for the readme. lgtm

File: README.md [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
-## Maintainability
 [![Maintainability](https://api.codeclimate.com/v1/badges/88c3d9e8784c8fa76af6/maintainability)](https://codeclimate.com/github/HumanLearning2021/HumanLearningApp/maintainability)
 
-## Test Coverage
 [![Test Coverage](https://api.codeclimate.com/v1/badges/88c3d9e8784c8fa76af6/test_coverage)](https://codeclimate.com/github/HumanLearning2021/HumanLearningApp/test_coverage)
+
+## The application
+Human Learning is an application whose objective is to facilitate learning of abstraction by matching images. There are two main aspects to the application: learning on a dataset and managing datasets. Traditionnaly one would use this learning methods with printed images, but this takes a lot of time to prepare and update. Here you can do all of this digitally.
+
+The application's database is organized in datasets who contain categories. Pictures are assigned to categories and each category has a picture which is set as it's representative.
```
Comment (Nitwix) [inline README.md]:
```suggestion
The application's database is organized in datasets who contain categories. Pictures are assigned to categories and each category has a picture which is set as its representative.
```

**Review by Nitwix — APPROVED** [review]
Good README, very clear

**Comment by Nitwix** [comment]
perfect, thanks!

**Pull request merged**